src/components/timeline/RowHeaderActivityTree.svelte

```diff
@@ -79,8 +88,8 @@
         style:height={`${rowHeight}px`}
         class="row-header-activity-group leaf st-button tertiary"
         class:selected={directive?.id === selectedActivityDirectiveId || span?.span_id === selectedSpanId}
-        on:dblclick={e => onLeafClick(e, node, 'dblClick')}
-        on:click={e => onLeafClick(e, node, 'mouseDown')}
+        on:dblclick={e => onLeafDblClick(e)}
```

Comment:
Reason for the change to click/dblclick handling?

Reply:
It appears that they required slightly different handling (`dblClick` vs. `mouseDown`) depending on what was selected. For example, double clicking a leaf should dispatch `selectedSpan/EventId`, whereas single clicking a leaf should dispatch whatever was selected.